### PR TITLE
livedl: init at unstable-2021-05-16

### PR DESCRIPTION
**Pull request by wakira**
###### Motivation for this change
livedl is a command-line tool written in Go to download nicovideo.jp livestreams

https://github.com/himananiito/livedl

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by wakira** [comment]
Result of `nixpkgs-review pr 127562` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>livedl</li>
  </ul>
</details>

File: pkgs/tools/misc/livedl/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+
```
Comment (musfay) [inline pkgs/tools/misc/livedl/default.nix:4]:
```suggestion
```

File: pkgs/tools/misc/livedl/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  pname = "livedl";
+  version = "a8720f1";
```
Comment (musfay) [inline pkgs/tools/misc/livedl/default.nix:6]:
```suggestion
  version = "unstable-2021-05-16";
```

File: pkgs/tools/misc/livedl/default.nix [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+  src = fetchFromGitHub {
+    owner = "himananiito";
+    repo = "livedl";
```
Comment (musfay) [inline pkgs/tools/misc/livedl/default.nix:10]:
```suggestion
    repo = pname;
```

File: pkgs/tools/misc/livedl/default.nix [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    rev = "a8720f1e358e5b0ade6fdeb8aacc00781e6cc504";
+    sha256 = "1zax215jp6sl47m8ahssyyrbzn96dh74srq9g61jc76sq10xg329";
+  } + "/src";
```
Comment (musfay) [inline pkgs/tools/misc/livedl/default.nix:13]:
```suggestion
  };
```

File: pkgs/tools/misc/livedl/default.nix [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
```
Comment (musfay) [inline pkgs/tools/misc/livedl/default.nix:14]:
```suggestion
  sourceRoot = "source/src";
```

**Review by musfay — CHANGES_REQUESTED** [review]

**Comment by wakira** [comment]
Result of `nixpkgs-review pr 127562` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>livedl</li>
  </ul>
</details>

**Comment by musfay** [comment]
@wakira PR name should be `livedl: init at unstable-2021-05-16`

**Review by musfay — APPROVED** [review]

**Pull request merged**